axlearn/common/array_serialization.py

```diff
@@ -596,7 +596,7 @@ async def _run_deserializer():
             byte_limiter = serialization._LimitInFlightBytes(concurrent_bytes)
             h2d_limiter = serialization._LimitInFlightBytes(_get_premapped_buffer_size())
 
-            future_arrays = jax.tree_util.tree_map(
+            future_arrays = jax.tree.map(
```

Comment:
Is `jax.tree.map` preferred over `jax.tree_util.tree_map`? I saw below message on my test, but it is not clear which one is preferred 

```
>>> import jax
>>> jax.tree.map(lambda x: x + 1, {"x": 7, "y": 42})
{'x': 8, 'y': 43}
>>> jax.tree_map(lambda x: x + 1, {"x": 7, "y": 42})
<stdin>:1: DeprecationWarning: jax.tree_map is deprecated: use jax.tree.map (jax v0.4.25 or newer) or jax.tree_util.tree_map (any JAX version).
{'x': 8, 'y': 43}
>>> 

```

Reply:
https://docs.jax.dev/en/latest/changelog.html#jax-0-4-26-april-3-2024 says
> `jax.tree_map()` is deprecated; use `jax.tree.map` instead, or for backward compatibility with older JAX versions, use [`jax.tree_util.tree_map()`](https://docs.jax.dev/en/latest/_autosummary/jax.tree_util.tree_map.html#jax.tree_util.tree_map).

sounds like a mild preference for `jax.tree.map`, but I'm not aware of it being a big deal (unless you need to support JAX older than 0.4.25)